### PR TITLE
Vm logging admin feature

### PR DESCRIPTION
**Pull request by sei-chershberger**

### CODE REVIEW

**Review by sei-tspencer — APPROVED** [review]

File: src/app/components/vm-usage-logging/vm-usage-logging.component.html [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+      #formDirective="ngForm"
+    >
+      <mat-form-field class="session-full-width" appearance="fill">
+        <mat-label>Log Name</mat-label>
+        <input
+          type=""
+          matInput
+          [formControl]="sessionNameFormControl"
+          [errorStateMatcher]="matcher"
+          placeholder="Ex. Exercise RCC-E MC1 2022- Blue and Red Teams"
```
Comment (sei-aschlackman) [inline src/app/components/vm-usage-logging/vm-usage-logging.component.html:22]: Might want a more generic example name here?

File: src/app/components/vm-main/vm-main.component.ts [hunk 2]
```diff
@@ -71,16 +84,41 @@ export class VmMainComponent implements OnInit, OnDestroy {
...
+
+    this.currentUser$ = this.authService.user$.pipe(
+      switchMap((u) => {
+        return this.userService.getUser(u.profile.sub);
+      })
+    );
+
+    this.loggingEnabled$ = this.vmUsageLoggingSessionService.getIsLoggingEnabled();
+
+    this.teamService.GetAllMyTeams(this.viewId ).pipe(take(1)).subscribe(tms => {
```
Comment (sei-aschlackman) [inline src/app/components/vm-main/vm-main.component.ts:96]: Might be easier/more efficient to use the /permissions endpoint to get the user's permissions by viewId in the future

File: swagger.json [hunk 3]
```diff
@@ -0,0 +1,3445 @@
+{
```
Comment (sei-aschlackman) [inline swagger.json:1]: I don't think we want this file in the repo.

**Review by sei-aschlackman — CHANGES_REQUESTED** [review]

File: src/app/components/vm-usage-logging/vm-usage-logging.component.html (hunk 1 above)
Comment (sei-chershberger) [reply src/app/components/vm-usage-logging/vm-usage-logging.component.html:22]: Done

File: swagger.json (hunk 3 above)
Comment (sei-chershberger) [reply swagger.json:1]: Removed.  Oops

**Review by sei-aschlackman — APPROVED** [review]

**Pull request merged**